### ISSUE TITLE
Wakatime not recording stats with latest vscode update.

### ISSUE THREAD

**Issue by nohupped** [wakatime/vscode-wakatime#155]
Hello,
I've been using wakatime for a long time, but since the last update of vscode, it appears to stop working. 

Initially, on the statusbar, it was stuck at 
![wakatime_initialisingpng](https://user-images.githubusercontent.com/8809560/90992483-a9a8c100-e5e2-11ea-872b-87ea256a9c47.png) forever.

I tried re-initializing it by deleting everything 

```
rm -rf ~/.wakatime*
```
and reinstalling the plugin.

This time, it initialized, but the stats are not recording in the dashboard.

These are the errors that I see in my vscode Developer Console.

![zip_header](https://user-images.githubusercontent.com/8809560/90993001-9e0ac980-e5e5-11ea-9c60-5bbcf8bd51f7.png)

![initialized](https://user-images.githubusercontent.com/8809560/90993003-a4994100-e5e5-11ea-83b9-99cf7da4a991.png)

```
Downloading wakatime-cli standalone...
```
happens every time I open the vscode.

 
Attaching the details of my Vscode/plugin/OS version

```
Version: 1.48.1
Commit: 3dd905126b34dcd4de81fa624eb3a8cbe7485f13
Date: 2020-08-19T17:11:59.963Z
Electron: 7.3.2
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Linux x64 5.8.3-arch1-1
```
Wakatime installed plugin version: latest at the time of reporting this.

```
v4.0.3
```

Please let me know if I am missing anything.

Thank you very much.

**Comment by alanhamlett** [comment]
After uninstalling you should run `rm -r ~/.vscode/extensions/wakatime*` since `~/.wakatime*` is your config and offline heartbeats db which shouldn't be deleted in case there were heartbeats still waiting to sync up to the api. The `wakatime-cli` dependency is extracted into `~/.vscode/extensions/wakatime*` so removing that folder resets the plugin.

Is this bug fixed if you install the old version 4.0.0?
<img width="436" alt="Screen Shot 2020-08-23 at 6 44 29 PM" src="https://user-images.githubusercontent.com/522344/90995409-c0163480-e570-11ea-8fe4-b3cf74ed9455.png">

**Comment by jeffryang24** [comment]
After uninstalled it and run `rm -r ~/.vscode/extensions/wakatime*`, still have the above error. Especially these lines still appear:

- [WakaTime][ERROR] Error: Invalid or unsupported zip format. No END header found
- [WakaTime][WARN] Error: ENOENT: no such file or directory, chmod '/home/jeffryangtoni/.vscode/extensions/wakatime.vscode-wakatime-4.0.3/wakatime-cli/wakatime-cli'

:bow:

**Comment by alanhamlett** [comment]
How about when installing the older version 4.0.0?

**Comment by alanhamlett** [comment]
Fixed with v4.0.4 by reverting back to request library.

**Comment by nohupped** [comment]
Thank you very much @alanhamlett. I checked it again by installing the older version `4.0.0` and it is working perfectly, however, the latest version (`v4.0.5`) is stuck at 

![wakatime_initialisingpng](https://user-images.githubusercontent.com/8809560/90992483-a9a8c100-e5e2-11ea-872b-87ea256a9c47.png)

since around last 20 minutes, and there is no error or warning in the Debug console on vscode. 
I've tried removing the extension completely by `rm -r ~/.vscode/extensions/wakatime*` and started fresh but it is still stuck there in the statusbar.

Is there anything else that I am missing? 

Thank you
